backend/db/manage_db.py

```diff
@@ -0,0 +1 @@
+# TODO create the database for the app
```

Comment:
did we ever finalize a database type? if we're doing postgres i can possibly have something up and running next week

Reply:
We'll go with SQLite3, which is basically the same as postgres. [Here's](https://flask.palletsprojects.com/en/1.1.x/tutorial/database/) the Flask page about how to set up the database.